core/src/test/java/org/infinispan/jmx/ClusterCacheStatsMBeanTest.java

```diff
@@ -54,10 +54,6 @@ public void testClusterStats() throws Exception {
       //sleep so we pick up refreshed values after remove
       TestingUtil.sleepThread(AbstractClusterStats.DEFAULT_STALE_STATS_THRESHOLD + 1000);
 
-      assertAttributeValueGreaterThanOrEqualTo(mBeanServer, clusterStats, "AverageWriteTime", 1);
-      assertAttributeValueGreaterThanOrEqualTo(mBeanServer, clusterStats, "AverageReadTime", 1);
-      assertAttributeValueGreaterThanOrEqualTo(mBeanServer, clusterStats, "AverageRemoveTime", 1);
```

Comment:
Could you keep this with 0 instead of 1?

Ideally I'd like to use the controlled time service and use `assertEquals`, but `>= 0` is good enough for now.